src/local/LocalTransport.ts

```diff
@@ -0,0 +1,93 @@
+import assert from 'assert';
+import _, { values, valuesIn } from 'lodash';
+import debug from 'debug';
+import Matcher from './matcher';
+import AutoBus from '../AutoBus';
+
+const log = debug('autobus:transport:local');
+
+export interface Listener<T> {
+  (message: T): void;
+}
+
+export default class LocalTransport {
+  _bus: AutoBus;
+  _wildcardMatchers: Matcher;
+  _channelListeners: Record<string, Listener<any>[]>;
+
+  constructor(bus: AutoBus) {
+    this._bus = bus;
+    this._channelListeners = {};
+    this._wildcardMatchers = new Matcher();
+  }
+
+  send<T>(channel: string, data: T): void {
+    const self = this;
```

Comment:
This should not be needed. Replace `function` callbacks with arrow functions and use the regular `this` instead.